### PR TITLE
don't execute the action on click if the button is disabled

### LINKED ISSUE #7: Don't send action when button is disabled
Hi, thank you for this wrapper around Ladda 👍 

What do you think of checking the disable state of the button before sending this action ? 
Well, maybe I'm just making a mistake somewhere, but it seems like I can for example click on the button whereas the action is already processed.

### PR DESCRIPTION
**Pull request by sly7-7**
Fixes #7

### CODE REVIEW

**Comment by sly7-7** [comment]
... my bad for prettier mess... I update it

File: addon/components/ladda-button.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default Component.extend({
     '_spinnerColor:data-spinner-color',
     '_spinnerLines:data-spinner-lines',
     '_spinnerSize:data-spinner-size',
-    'type'
+    'type',
```
Comment (deanylev) [inline addon/components/ladda-button.js]: Not personally a fan of trailing commas, same on line 96

File: addon/components/ladda-button.js [hunk 2]
```diff
@@ -77,6 +77,9 @@ export default Component.extend({
   },
 
   click() {
+    if (this._disabled) {
+      return;
+    }
```
Comment (deanylev) [inline addon/components/ladda-button.js:82]: Newline here would be nice :)

File: tests/integration/components/ladda-button-test.js [hunk 3]
```diff
@@ -23,4 +24,21 @@ module('Integration | Component | ladda-button', function(hooks) {
 
     assert.equal(this.element.textContent.trim(), 'template block text');
   });
+
+  test("it does'nt send the action if already running", async function (assert) {
```
Comment (deanylev) [inline tests/integration/components/ladda-button-test.js]: Spelling mistake here, should be `doesn't`

File: tests/integration/components/ladda-button-test.js [hunk 4]
```diff
@@ -23,4 +24,21 @@ module('Integration | Component | ladda-button', function(hooks) {
...
+    assert.expect(1);
+    this.set('execute', () => {
+      assert.ok(true, 'action is called');
+      return Promise.resolve();
+    });
```
Comment (deanylev) [inline tests/integration/components/ladda-button-test.js]: Could make this function `async` to remove the `return`, not too fussed though

**Comment by sly7-7** [comment]
@deanylev I hope it's fine now :). Maybe you could add a .prettierrc or something like that ? (editorconfig doesn't have a setting for trailing coma for example). I'm using vscode as editor, with prettier installed, and when it sees an editorconfig file, it takes precedence over vscode config. I added a .prettierrc file locally to fix that.

**Comment by deanylev** [comment]
👍  been just me up until now so haven't worried about that sort of stuff but yeah a linter of some kind is a good idea

**Review by deanylev — APPROVED** [review]

**Pull request merged**